src/Translation/ThemeAwareTranslator.php

```diff
@@ -17,9 +17,11 @@
 use Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface;
 use Symfony\Component\Translation\MessageCatalogueInterface;
 use Symfony\Component\Translation\TranslatorBagInterface;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\LocaleAwareInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Component\Translation\TranslatorInterface as OldTranslatorInterface;
```

Comment:
Wrong order of uses

Reply:
Which order should I use ?

Reply:
You should add `Symfony\Contracts\...` after `use Symfony\Component\...`

Reply:
Could we call it `Legacy` instead of `Old`? This is the way Symfony calls it in their codebase.

Reply:
No worries about imports order, it'd be fixed automatically before release by the coding standard tool.